ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -89,24 +102,47 @@ export default class ExperimentalTab extends PureComponent {
           className="settings-page__content-row settings-page__content-row-experimental"
         >
           <div className="settings-page__content-item">
-            <span>{t('securityAlerts')}</span>
+            <Text
+              variant={TextVariant.bodyMd}
+              color={TextColor.textAlternative}
+            >
+              {t('securityAlerts')}
+            </Text>
             <div className="settings-page__content-description">
+              <Text variant={TextVariant.bodySm}>
+                {t('securityAlertsDescription')}
+              </Text>
               <Text
-                variant={TextVariant.bodySm}
+                variant={TextVariant.bodySmBold}
                 color={TextColor.textAlternative}
+                marginTop={4}
               >
-                {t('securityAlertsDescription')}
+                {t('preferredProvider')}
               </Text>
               <div
                 data-testid="settings-toggle-security-alert-blockaid"
                 className="settings-page__content-item-col settings-page__content-item-col__security-toggle-option"
               >
-                <Text
-                  variant={TextVariant.bodyMd}
-                  color={TextColor.textDefault}
-                >
-                  {t('blockaid')}
-                </Text>
+                <div>
+                  <Box display={Display.Flex}>
+                    <Text
+                      variant={TextVariant.bodyMd}
+                      color={TextColor.textDefault}
+                    >
+                      {t('blockaid')}
+                    </Text>
+                    <Tag marginLeft={2} label="Recommended" />
```

Comment:
I think marginLeft will not work with right-to-left languages here. have you tested an right-to-left language by chance? I think we'll want to use `margin-inline-end` or `padding-inline-end` here instead

Reply:
Seems to work well in RTL:

![Screenshot 2023-10-02 at 3 21 23 PM](https://github.com/MetaMask/metamask-extension/assets/2182307/7da3c9a7-c6a5-4d71-8263-190af043c86e)

Reply:
Oh, nice! Thanks for checking